tests/shared/importers/test_rasa.py

```diff
@@ -45,13 +45,13 @@ async def test_read_conversation_tests(project: Text):
         training_data_paths=[str(Path(project) / DEFAULT_CONVERSATION_TEST_PATH)]
     )
 
-    test_stories = await importer.get_conversation_tests()
+    test_stories = importer.get_conversation_tests()
     assert len(test_stories.story_steps) == 7
 
 
 async def test_rasa_file_importer_with_invalid_config():
```

Comment:
as well as `test_read_conversation_tests` and `test_rasa_file_importer_with_invalid_domain`
```suggestion
def test_rasa_file_importer_with_invalid_config():
```